### PR TITLE
Test to ensure that page refresh doesn't affect url bar focus

### PR DESCRIPTION
**Pull request by bwaresiak**
Task/Issue URL: https://app.asana.com/0/0/1207004088983506/f
Tech Design URL:
CC:

**Description**:

Add test to ensure URL bar doesn't lose focus on site refresh.

**Steps to test this PR**:

Run the tests to make sure these pass.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: .maestro/browser_features/search_bar.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# search_bar.yaml
+appId: com.duckduckgo.mobile.ios
+tags:
+    - release
```
Comment (loremattei) [inline .maestro/browser_features/search_bar.yaml:4]: The initial plan was to keep tags aligned to folders to make it clearer what was running at any point. It looks like we've already lost this over time, so this is likely fine at this point.

File: .maestro/shared/delay.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+appId: com.duckduckgo.mobile.ios
+---
+
+- runFlow:
+    when:
+      visible:
+        text: "This is just a text that will be never matched so this invocation will cause 5s delay"
+        index: 0
+    file: onboarding_browsing.yaml
```
Comment (loremattei) [inline .maestro/shared/delay.yaml]: Anything is fine here obviously. I'm just wondering if something like
```
commands: 
  - stopApp: appId
```

would make intentions clearer here.

**Review by loremattei — APPROVED** [review]
Left a small and very minor suggestion. 
LGTM!
Thanks for adding this!

File: .maestro/shared/delay.yaml (hunk 2 above)
Comment (bwaresiak) [reply .maestro/shared/delay.yaml]: Thanks, I've tweaked the command

**Pull request merged**